content/gettingstarted/index.adoc

```diff
@@ -52,7 +52,11 @@ icon:bullhorn[size=1x]{nbsp} This script requires `envsubst` installed and in yo
 bash <(curl -L http://git.io/getLatestKiali)
 ----
 
-By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKiali[the comments at the top of the install script].
+By default, the installation script will prompt you to enter the login option you wish to use. Choosing the 'login' option will secure Kiali using a username and password. The script will also prompt you to enter the username and passphrase to use. Choosing the 'anonymous' option will remove authentication from Kiali and it will be fully open to anyone without requiring credentials. Choosing the 'openshift' option will use the OpenShift OAuth login and will provide a access based on the user's permission in OpenShift.
```

Comment:
* Is it "password" or "passphrase", you use both in this blurb.
* "provide a access" -> "provide access"
* "user's permission in OpenShift" -> "user's OpenShift roles"

Reply:
We should be using the word "passphrase" when we are talking about Kiali's login credentials. That is what all upstream Istio docs and all our kiali settings use.